src/Validation.elm

```diff
@@ -165,60 +159,79 @@ field value =
 
 
 {-| Default setting of Field with `Valid a` validity.
+For `Field String String` use an identity function.
 
     import Validation exposing (Field, field)
 
     intValue : Field String Int
     intValue =
-        preValidatedField "50"
+        preValidatedField String.fromInt 50
 
 
     -- Field "50" (Valid 50)
 
 -}
-preValidatedField : val -> Field String val
-preValidatedField value =
-    Field (toString value) (Valid value)
+preValidatedField : (val -> String) -> val -> Field String val
+preValidatedField fn value =
+    Field (fn value) (Valid value)
```

Comment:
Nebude lepsi tuhle fci zrusi a udelat misto ni neco jako
```haskell
preValidIntField : Int -> Field String Int
preValidIntField val =
    Field (String.fromInt val) (Valid val)
```
obdobne pak pro `Float, Bool, String,...`?

Reply:
Nn, takto je to dobre obecne.